slices/libnss-extrausers.yaml

```diff
@@ -0,0 +1,21 @@
+package: libnss-extrausers
+
+essential:
+  - libnss-extrausers_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libnss-extrausers_var
+    contents:
+      /usr/lib/libnss_extrausers.so.2:
+      /usr/lib32/libnss_extrausers.so.2: { arch: [amd64, s390x] }
```

Comment:
Let's be consistent with the other file with regards to the trailing version number in the path, `.2` in this case. Let's use globs like below, if they can change.